sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormatSuite.scala

```diff
@@ -129,6 +131,20 @@ abstract class ParquetFileFormatSuite
       }
     }
   }
+
+  test("Write and read back TIME values") {
```

Comment:
Move it here from `ParquetIOSuite` to test for both V1 and V2 datasources, and avoid the test duplication.